sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -290,3 +290,24 @@ ALTER TABLE `insurance_companies` CHANGE `inactive` `inactive` TINYINT(1) NOT NU
 UPDATE `layout_options` SET `fld_length` = '8' WHERE `layout_options`.`form_id` = 'DEM' AND `layout_options`.`field_id` = 'postal_code';
 #EndIf
 
+#IfNotColumnType form_observation date datetime
```

Comment:
in database.sql date is already datetime, so this block will be skipped
will separate this out into separate if blocks
also need to add new date_end field here

Reply:
disregard above comment, i messed up. this block looks good :)